packages/atomic/src/components/search/generated-answer/atomic-generated-answer-feedback/atomic-generated-answer-feedback-modal.tsx

```diff
@@ -0,0 +1,275 @@
+import {GeneratedAnswer} from '@coveo/headless';
+import {GeneratedAnswerFeedback} from '@coveo/headless/dist/definitions/features/generated-answer/generated-answer-analytics-actions';
+import {
+  Component,
+  State,
+  h,
+  Watch,
+  Prop,
+  Element,
+  Event,
+  EventEmitter,
+} from '@stencil/core/internal';
+import CloseIcon from '../../../../images/close.svg';
+import Success from '../../../../images/success.svg';
+import {
+  InitializableComponent,
+  InitializeBindings,
+} from '../../../../utils/initialization-utils';
+import {updateBreakpoints} from '../../../../utils/replace-breakpoint';
+import {once, randomID} from '../../../../utils/utils';
+import {Button} from '../../../common/button';
+import {Bindings} from '../../atomic-search-interface/atomic-search-interface';
+
+@Component({
```

Comment:
Absolutely needs to be marked as internal so it doesn't get documented before the GA. It's the case for all our RGA components.

```suggestion
/**
 * @internal
 */
@Component({
```